src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started its work.");

```

Comment:
The message in the `doWork` method should indicate the type of `Machine` more informatively. Consider specifying that it is a `Bulldozer` that has started its work. For example: "Bulldozer started to move earth."